org.metaborg.sdf2table/src/main/java/org/metaborg/sdf2table/grammar/CharacterClass.java

```diff
@@ -82,8 +81,7 @@ public CharacterClass(BitSet chars, boolean eof) {
     }
 
     @Override public IStrategoTerm toAtermList(ITermFactory tf) {
-        List<IStrategoTerm> terms = new ArrayList<>();
-
+        IStrategoList.Builder terms = tf.arrayListBuilder(chars.size());
```

Comment:
For the full character class range (0-1114111), `chars.size() == 1114112` while there is only one range. This means that the backing array will be _very_ sparse. Can the initial size of the `arrayListBuilder` be left out here?